### PR TITLE
docs: fix `PHPStan` in `Commands`

### PR DESCRIPTION
**Pull request by ddevsr**
**Description**
See #7731

**Checklist:**
- [x] Securely signed commits
- [x] Component(s) with PHPDoc blocks, only if necessary or adds value
- [ ] Unit testing, with >80% coverage
- [ ] User guide updated
- [x] Conforms to style guide

### CODE REVIEW

File: system/Commands/Database/ShowTableInfo.php [hunk 1]
```diff
@@ -188,6 +188,13 @@ private function restoreDBPrefix(): void
         $this->db->setPrefix($this->DBPrefix);
     }
 
+    /**
+     * Show Data of Table
+     *
+     * @used-by run
```
Comment (kenjis) [inline system/Commands/Database/ShowTableInfo.php:194]:
```suggestion
     * @used-by run()
```

File: system/Commands/Database/ShowTableInfo.php [hunk 2]
```diff
@@ -207,6 +214,15 @@ private function showDataOfTable(string $tableName, int $limitRows, int $limitFi
         CLI::table($this->tbody, $thead);
     }
 
+    /**
+     * Show All Tables
+     *
+     * @used-by run
```
Comment (kenjis) [inline system/Commands/Database/ShowTableInfo.php:220]:
```suggestion
     * @used-by run()
```

File: system/Commands/Database/ShowTableInfo.php [hunk 3]
```diff
@@ -207,6 +214,15 @@ private function showDataOfTable(string $tableName, int $limitRows, int $limitFi
...
+     *
+     * @param array<int, string> $tables
```
Comment (kenjis) [inline system/Commands/Database/ShowTableInfo.php:222]:
```suggestion
     * @param list<string> $tables
```

File: system/Commands/Database/ShowTableInfo.php [hunk 4]
```diff
@@ -219,6 +235,13 @@ private function showAllTables(array $tables)
         CLI::newLine();
     }
 
+    /**
+     * @used-by showAllTables
```
Comment (kenjis) [inline system/Commands/Database/ShowTableInfo.php:239]:
```suggestion
     * @used-by showAllTables()
```

File: system/Commands/Database/ShowTableInfo.php [hunk 5]
```diff
@@ -219,6 +235,13 @@ private function showAllTables(array $tables)
...
+     *
+     * @param array<int, string> $tables
```
Comment (kenjis) [inline system/Commands/Database/ShowTableInfo.php:241]:
```suggestion
     * @param list<string> $tables
```

File: system/Commands/Database/ShowTableInfo.php [hunk 6]
```diff
@@ -244,6 +267,11 @@ private function makeTbodyForShowAllTables(array $tables): array
         return $this->tbody;
     }
 
+    /**
+     * @used-by showDataOfTable
```
Comment (kenjis) [inline system/Commands/Database/ShowTableInfo.php:271]:
```suggestion
     * @used-by showDataOfTable()
```
By the way, do we really need `@used-by` like this?

**Comment by kenjis** [comment]
@ddevsr 
I use the label [documentation](https://github.com/codeigniter4/CodeIgniter4/labels/documentation) to PRs to change the User Guide.
Do you think it is better to use the label to the PR to doc comments like this?

File: system/Commands/Database/ShowTableInfo.php (hunk 6 above)
Comment (kenjis) [reply system/Commands/Database/ShowTableInfo.php:271]: Can you remove all `@used-by`?
I think we don't need `@used-by` when we can jump to callers in an IDE.

**Review by datamweb — APPROVED** [review]

**Review by kenjis — APPROVED** [review]

**Comment by kenjis** [comment]
@ddevsr Thank you!

**Pull request merged**